### PR TITLE
Fix an issue of yaml and json format

### PR DESCRIPTION
**Pull request by trungnvfet**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fix some wrong json and yaml format in project
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: vendor/github.com/spf13/viper/README.md [hunk 1]
```diff
@@ -500,7 +500,7 @@ Extract sub-tree from Viper.
 
 For example, `viper` represents:
 
-```json
+```yaml
```
Comment (thockin) [inline vendor/github.com/spf13/viper/README.md]: We don't hand-edit vendor/ files.  You'd have to fix that upstream

**Comment by trungnvfet** [comment]
@thockin Thank you. I just pushed new PR.

**Pull request merged**